### PR TITLE
Updating test apps for CSH backend

### PR DESCRIPTION
**Pull request by doudounan**

### CODE REVIEW

File: GoogleDataTransportCCTSupport/GDTCCTTestApp/app.swift [hunk 1]
```diff
@@ -41,15 +42,46 @@ import GoogleDataTransport
...
+            case 1:
+                theTransport = fllTransport;
+                break;
+                
+            case 2:
+                theTransport = cshTransport;
+                break;
+                
+            default:
+                theTransport = cctTransport;
```
Comment (mikehaney24) [inline GoogleDataTransportCCTSupport/GDTCCTTestApp/app.swift]: add break

File: GoogleDataTransportCCTSupport/GDTCCTTestApp/app.swift [hunk 2]
```diff
@@ -41,15 +42,46 @@ import GoogleDataTransport
...
+        case 1:
+            theTransport = fllTransport;
+            break;
+            
+        case 2:
+            theTransport = cshTransport;
+            break;
+            
+        default:
+            theTransport = cctTransport;
```
Comment (mikehaney24) [inline GoogleDataTransportCCTSupport/GDTCCTTestApp/app.swift]: add break

File: GoogleDataTransportCCTSupport/GDTCCTTestApp/app.swift [hunk 3]
```diff
@@ -41,15 +42,46 @@ import GoogleDataTransport
       if !Thread.current.isMainThread {
         DispatchQueue.main.sync {
           if Globals.IsMonkeyTesting {
-            backendSwitch?.selectedSegmentIndex = Int(arc4random_uniform(2))
+            backendSwitch?.selectedSegmentIndex = Int(arc4random_uniform(3))
           }
-          theTransport = backendSwitch?.selectedSegmentIndex == 0 ? cctTransport : fllTransport
+            switch(backendSwitch?.selectedSegmentIndex){
```
Comment (mikehaney24) [inline GoogleDataTransportCCTSupport/GDTCCTTestApp/app.swift]: nit: space between paren and curly brace

File: GoogleDataTransportCCTSupport/GDTCCTTestApp/app.swift [hunk 4]
```diff
@@ -41,15 +42,46 @@ import GoogleDataTransport
...
+                theTransport = cctTransport;
+            }
         }
       } else {
         if Globals.IsMonkeyTesting {
-          backendSwitch?.selectedSegmentIndex = Int(arc4random_uniform(2))
+          backendSwitch?.selectedSegmentIndex = Int(arc4random_uniform(3))
+        }
+        
+        switch(backendSwitch?.selectedSegmentIndex){
```
Comment (mikehaney24) [inline GoogleDataTransportCCTSupport/GDTCCTTestApp/app.swift]: space between paren and curly brace

File: GoogleDataTransportCCTSupport/GDTCCTTestApp/app.swift [hunk 5]
```diff
@@ -76,15 +109,45 @@ import GoogleDataTransport
       if !Thread.current.isMainThread {
         DispatchQueue.main.sync {
           if Globals.IsMonkeyTesting {
-            backendSwitch?.selectedSegment = Int(arc4random_uniform(2))
+            backendSwitch?.selectedSegment = Int(arc4random_uniform(3))
+          }
+          switch(backendSwitch?.selectedSegment){
```
Comment (mikehaney24) [inline GoogleDataTransportCCTSupport/GDTCCTTestApp/app.swift]: space between paren and curly brace

File: GoogleDataTransportCCTSupport/GDTCCTTestApp/app.swift [hunk 6]
```diff
@@ -76,15 +109,45 @@ import GoogleDataTransport
...
+              
+          case 1:
+              theTransport = fllTransport;
+              break;
+              
+          case 2:
+              theTransport = cshTransport;
+              break;
+              
+          default:
```
Comment (mikehaney24) [inline GoogleDataTransportCCTSupport/GDTCCTTestApp/app.swift]: needs break

File: GoogleDataTransportCCTSupport/GDTCCTTestApp/app.swift [hunk 7]
```diff
@@ -76,15 +109,45 @@ import GoogleDataTransport
...
+              theTransport = cctTransport;
           }
-          theTransport = backendSwitch?.selectedSegment == 0 ? cctTransport : fllTransport
         }
       } else {
         if Globals.IsMonkeyTesting {
-          backendSwitch?.selectedSegment = Int(arc4random_uniform(2))
+          backendSwitch?.selectedSegment = Int(arc4random_uniform(3))
+        }
+        switch(backendSwitch?.selectedSegment){
```
Comment (mikehaney24) [inline GoogleDataTransportCCTSupport/GDTCCTTestApp/app.swift]: space between paren and curly brace

File: GoogleDataTransportCCTSupport/GDTCCTTestApp/app.swift [hunk 8]
```diff
@@ -76,15 +109,45 @@ import GoogleDataTransport
...
+        case 1:
+            theTransport = fllTransport;
+            break;
+            
+        case 2:
+            theTransport = cshTransport;
+            break;
+            
+        default:
+            theTransport = cctTransport;
```
Comment (mikehaney24) [inline GoogleDataTransportCCTSupport/GDTCCTTestApp/app.swift]: break

File: GoogleDataTransportCCTSupport/GDTCCTTestApp/app.swift (hunk 1 above)
Comment (doudounan) [reply GoogleDataTransportCCTSupport/GDTCCTTestApp/app.swift]: fixed

**Comment by maksymmalyhin** [comment]
FYI, you may use `./scripts/style.sh` to autocorrect most of the formatting errors (see https://github.com/firebase/firebase-ios-sdk#code-formatting for more details)

**Pull request merged**